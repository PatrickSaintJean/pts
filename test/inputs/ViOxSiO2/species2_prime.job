#!/bin/bash
#$-o /home/hugh/src/sem-global-ts/inputs/ViOxSiO2/species2_prime.out -j y
#$-N species2_prime
#$-S /bin/bash
#$-pe shm 4
#$-M  hugh.chaffey-millar@ch.tum.de
. /etc/profile

time g03 < /home/hugh/src/sem-global-ts/inputs/ViOxSiO2/species2_prime.com > /home/hugh/src/sem-global-ts/inputs/ViOxSiO2/species2_prime.log

