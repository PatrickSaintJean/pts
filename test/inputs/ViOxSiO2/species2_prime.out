
real	80m49.581s
user	311m1.986s
sys	1m4.480s

real	215m7.739s
user	504m38.508s
sys	1m6.268s
